analytics_data_api/constants/educ_level.py

```diff
@@ -0,0 +1,16 @@
+UNKNOWN = 'unknown'
+
+LEVEL_OF_EDUCATION_LOOKUP = dict([
+    ('p', 'doctorate'),
+    ('m', 'masters'),  # Master's or professional degree
+    ('b', 'bachelors'),
+    ('a', "associate"),
+    ('hs', "secondary"),
+    ('jhs', "middle"),
+    ('el', "elementary"),
+    ('none', "none"),
+    ('other', "other"),
+    # Deprecated:
+    ('p_se', 'doctorate'),  # Doctorate in science or engineering
+    ('p_oth', 'doctorate'),  # Doctorate in another field
+])
```

Comment:
@bradenmacdonald nit: this could be a dict literal. Is there a reason it's converted from a list of tuples with `dict([...])`?